### DISCUSSION TITLE
Ödev 2 - Gönderilecek Video Sayısı

### DISCUSSION THREAD

**Discussion by denizkodas** [gusanmaz/BMB311_OS_2023#9] [Q&A] [answered]
Merhaba hocam, veri yapıları ödevi için verdiğiniz kitaptaki konuları incelediğimde 15 tane konu olduğunu gördüm. Bizim grubumuz 6 kişilik, ödev açıklamasında her grup üyesinin bir video çekmesini söylemişsiniz. Bizim 15 konuyu kendi aramızda paylaşıp videoları çekmemiz mi gerekiyor yoksa 6 kişi birer videodan toplam 6 video çekmememiz yeterli midir?

**Comment by gusanmaz** [comment, marked as answer]
Ödev açıklamalarında öyle bir talep bulunmamakta. Şu bilgiler var açıklamalarda:

> * Her grup kendi videosunu çekmelidir.
>  * Video linki tek bir videonun URL'si olabilecegi gibi, farkli konu basliklari icin olusturulmus kisa videolarin yer aldigi bir playlist'in URL'si olabilir.

Kısacası ister tek video, ister çok video içeren bir playlist hazırlayın tüm grup üyelerinin `threads.txt´ dosyasının en üst satırına tek ve aynı URL'yi yerleştirmesi gerekiyor.
